ormconfig.json

```diff
@@ -0,0 +1,22 @@
+{
```

Comment:
Is there a reason we want to have an ormconfig at the top level in addition to `backend/core`? I think it's probably better to keep everything there unless the tooling doesn't support it.

Reply:
ormconfig didn't play nicely with workspace config that we have. I think that might me a subject for a separate issue.

Reply:
Ok, I think I got this solved by deleting both ormconfig files and putting everything into .env and .env.template, which has the added benefit of less confusion when we deploy to heroku. @software-project are you ok if I commit those changes directly here?